commit f149e6f1daf34c284485254fa020bdd6d6838a2d
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Thu Oct 7 16:30:25 2010 -0700

    Staging: brcm80211: remove unused irq typedef
    
    If it's not needed, why define it?
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/linuxver.h
+++ b/drivers/staging/brcm80211/include/linuxver.h
@@ -33,9 +33,6 @@
 #include <asm/io.h>
 
 #include <linux/workqueue.h>
-
-typedef irqreturn_t(*FN_ISR) (int irq, void *dev_id, struct pt_regs * ptregs);
-
 #include <linux/sched.h>
 #include <linux/ieee80211.h>
 
